services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -176,6 +175,32 @@ const resolveDelegateInfo = async accounts => {
 				account.pomHeights = account.dpos.delegate.pomHeights
 					.sort((a, b) => b - a).slice(0, 5)
 					.map(height => ({ start: height, end: height + punishmentHeight }));
+
+				const [lastForgedBlock = {}] = await blocksDB.find({
+					generatorPublicKey: account.publicKey,
+					limit: 1,
+				});
+				account.dpos.delegate.lastForgedHeight = lastForgedBlock.height || null;
+
+				// Iff the COMPLETE blockchain is SUCCESSFULLY indexed
```

Comment:
```suggestion
				// If the COMPLETE blockchain is SUCCESSFULLY indexed
```

Reply:
@nagdahimanshu It is not a typo.`iff` is the acronym for `if and only if`.

Reply:
Thanks for clarification 👍